### PR TITLE
Improve Java track

### PR DESCRIPTION
**Pull request by mthmulders**
This pull request contains a lot of textual improvements for the Java track (and two minor ones for the .NET track).

@EdwinVW, please refrain from merging until @TomCools has reviewed the updated content.

### CODE REVIEW

**Comment by EdwinVW** [comment]
Changes in Assignment 6 are obsolete because if fix in PR #31. I suggest we merge PR #31 first and than update this PR.

File: Assignment02/step-by-step-java.md [hunk 1]
```diff
@@ -237,6 +237,8 @@ Now you'll change the code to use the Dapr-provided `DaprClient` to call the Veh
 
    > Also note that the `invokeMethod` method of the `DaprClient` returns a `Mono`. Because we don't want to get distracted by all the possibilities that reactive progamming brings, we immediately invoke `block` to await the result. In a real-world scenario, it would make more sense to propagate that Mono through our application to make it more reactive.
 
+   > At this point, you could consider removing the `vehicle-information.address` entry from the `application.yml` file. The Dapr SDK for Java does not need it.
```
Comment (TomCools) [inline Assignment02/step-by-step-java.md:240]: Should we make this more explicit and tell them to remove it? For the good cause? :)

**Comment by TomCools** [comment]
Besides that one comment, I approve this message :)

File: Assignment02/step-by-step-java.md (hunk 1 above)
Comment (mthmulders) [reply Assignment02/step-by-step-java.md]: Yep, done.

**Comment by mthmulders** [comment]
I've updated the step-by-step guide for Assignment 6 of the Java track to take #31 into account. @TomCools already approved of these changes, so please merge at your convenience, @EdwinVW  :-)

**Pull request merged**